packages/jest-util/src/createProcessObject.ts

```diff
@@ -5,6 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import {PassThrough} from 'stream';
```

Comment:
should we import from `readable-stream` instead to avoid inconsistencies between node versions?

Reply:
Yeah, we can do that, given that we're already depending on that module transitively.